CMakeLists.txt

```diff
@@ -330,7 +330,7 @@ cmake_dependent_option(
 set(MKLDNN_ENABLE_CONCURRENT_EXEC ${USE_MKLDNN})
 cmake_dependent_option(USE_MKLDNN_CBLAS "Use CBLAS in MKLDNN" OFF "USE_MKLDNN"
                        OFF)
-option(USE_STATIC_MKL "Prefer to link with MKL statically (Unix only)" OFF)
+option(USE_STATIC_MKL "Prefer to link with MKL statically (recommanded)." ON)
```

Comment:
rec911ended

Reply:
@ezyang not understand it. And I'm still fixing CI now.

Reply:
There is a typo. But also, we don't want static MKL, this is a deliberate decision